spec/features/openid_connect/openid_connect_spec.rb

```diff
@@ -137,11 +135,6 @@
       "http://www.example.com/openid_connect/logout?id_token_hint=#{id_token}",
     )
 
-    expect(page.response_headers['Content-Security-Policy']).to include(
-      'form-action \'self\' gov.gsa.openidconnect.test://result '\
-      'gov.gsa.openidconnect.test://result/signout',
-    )
-
```

Comment:
I have a gut feeling this is still needed....

My first thought is that a client app that receives the logout result may do a form submission back to itself immediately or something which is why we added this in https://github.com/18F/identity-idp/pull/3479? Definitely want to confirm it

Reply:
I think `form-action` only applies to forms submitted from `secure.login.gov`, and I talked with @achapm a bit about it and since we only redirect from our side without submitting forms on logout, it should be safe.